docker-compose.yml

```
@@ -737,6 +741,8 @@ services:
       service: microfrontend
     working_dir: '/edx/app/frontend-app-authn'
     container_name: "edx.${COMPOSE_PROJECT_NAME:-devstack}.frontend-app-authn"
+    environment:
+      PARAGON_BRAND_PACKAGE: '@edx/brand-edx.org@~2.0.0'
```

edx-internal shows this version as `2.x` currently.

```suggestion
      PARAGON_BRAND_PACKAGE: '@edx/brand-edx.org@2.x'
```

https://github.com/edx/edx-internal/blob/b3bd6adf8407b986349272f33b37922c620d8857/frontends/frontend-app-authn/prod_config.yml#L58